<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analysis</class>
 <widget class="QWidget" name="Analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gb_analysis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Analysis</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_analysis_name">
          <property name="text">
           <string>Analysis Name:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_analysis_name</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_edit_analysis_name"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_analysis_performed_by">
          <property name="text">
           <string>Analysis Performed by:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_analysis_performed_by</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="l_edit_analysis_performed_by"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_analysis_data">
          <property name="text">
           <string>Analysis date:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="date_analysis"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_solution">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Solution</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_fit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Fit Data</string>
          </property>
          <property name="icon">
           <iconset resource="icon/icon.qrc">
            <normaloff>:/menu_bar_icon/settings.png</normaloff>:/menu_bar_icon/settings.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_output">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_edit_output_transmissivity"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_output_storativity">
          <property name="text">
           <string>Storativity:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="l_edit_output_storativity"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_output_transmissivity">
          <property name="text">
           <string>Transmissivity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="btn_generate_report">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Generate Report</string>
          </property>
          <property name="icon">
           <iconset resource="icon/icon.qrc">
            <normaloff>:/menu_bar_icon/report.png</normaloff>:/menu_bar_icon/report.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="AnalysisCurve" name="wid_analysis_graph" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalysisCurve</class>
   <extends>QWidget</extends>
   <header>analysis_curve.py</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
